Examples/Makefile.in

```diff
@@ -324,7 +324,7 @@ d_run:
 # -----------------------------------------------------------------
 
 d_version:
-	($(DCOMPILER) --version 2> /dev/null || $(DCOMPILER)) | head -n 3
+	($(DCOMPILER) --version 2> /dev/null) | head -n 3
```

Comment:
We can lose the outer parentheses too now:

```
d_version:
	$(DCOMPILER) --version 2> /dev/null | head -n 3
```

Reply:
Sure